welcome/index.adoc

```diff
@@ -325,15 +325,13 @@ endif::openshift-rosa[]
 ifdef::openshift-enterprise[]
 == Hosted control plane activities
 
-* **Support for bare metal and {VirtProductName}**: {hcp-capital} for {product-title} is now Generally Available on bare metal and {VirtProductName} platforms. For more information, see the following documentation:
+* **Support for {aws-first}**: {hcp-capital} for {product-title} is now Generally Available on the {aws-short} platform. For more information, see the following documentation:
 
-** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-bm[Configuring hosted control plane clusters on bare metal]
-** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#hosted-control-planes-manage-kubevirt[Managing hosted control plane clusters on OpenShift Virtualization]
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosting cluster on {aws-short}]
```

Comment:
I get a 404 error for this link.

Reply:
Yep. As I mentioned in the first comment of this PR, none of the links will work until July 18. Thanks!